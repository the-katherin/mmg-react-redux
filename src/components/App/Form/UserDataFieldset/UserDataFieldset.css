.user-data-fieldset__inputs-grid{
    display: grid;
    grid-row-gap: 10px;
    justify-items: center;

}

.user-data-fieldset__inputs-grid  input{
    font-size: inherit;

}





