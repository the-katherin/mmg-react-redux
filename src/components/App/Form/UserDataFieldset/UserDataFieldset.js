import React from 'react';

const UserDataFieldset = () => (
    <fieldset className='user-data-fieldset' name='userData'>

        <legend>Fill the form below</legend>

        <div className="user-data-fieldset__inputs-grid">

            <div>
                <input type="text" id='userName' placeholder='Enter your first name' pattern='[A-Za-zА-Яа-яЁё]{1,15}' required />
            </div>

            <div>
                <input type="text" id="userLastName" placeholder='Enter your last name' pattern='[A-Za-zА-Яа-яЁё]{1,15}' required />
            </div>

            <div>
                <input type="email" id='email' placeholder='Enter your email' required />
            </div>

        </div>

    </fieldset>
);

export default UserDataFieldset;