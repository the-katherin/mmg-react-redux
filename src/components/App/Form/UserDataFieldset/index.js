import './UserDataFieldset.css';

export { default } from './UserDataFieldset';