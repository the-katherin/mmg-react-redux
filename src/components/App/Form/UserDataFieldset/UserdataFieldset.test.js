import React from "react";
import UserDataFieldset from "./UserDataFieldset";

describe("UserDataFieldset", () => {
    test("match snapshot", () => {
        const wrapper = shallow(<UserDataFieldset />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});