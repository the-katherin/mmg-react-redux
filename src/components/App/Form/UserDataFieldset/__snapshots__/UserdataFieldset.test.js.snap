// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDataFieldset match snapshot 1`] = `
<fieldset
  className="user-data-fieldset"
  name="userData"
>
  <legend>
    Fill the form below
  </legend>
  <div
    className="user-data-fieldset__inputs-grid"
  >
    <div>
      <input
        id="userName"
        pattern="[A-Za-zА-Яа-яЁё]{1,15}"
        placeholder="Enter your first name"
        required={true}
        type="text"
      />
    </div>
    <div>
      <input
        id="userLastName"
        pattern="[A-Za-zА-Яа-яЁё]{1,15}"
        placeholder="Enter your last name"
        required={true}
        type="text"
      />
    </div>
    <div>
      <input
        id="email"
        placeholder="Enter your email"
        required={true}
        type="email"
      />
    </div>
  </div>
</fieldset>
`;
